<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu Clientes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-8</x>
      <y>-1</y>
      <width>821</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/imagenes/fondoaplicacion.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableView" name="tablaclientes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>256</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnNuevo">
       <property name="text">
        <string>Nuevo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnModificar">
       <property name="text">
        <string>Modificar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>351</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Nombre</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtNombrecli">
           <property name="maxLength">
            <number>13</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>RTN</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtRtn">
         <property name="maxLength">
          <number>45</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Direccion</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtDireccion">
         <property name="maxLength">
          <number>45</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Telefono</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtTelefono">
         <property name="maxLength">
          <number>45</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtEmail">
         <property name="maxLength">
          <number>45</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnGuardar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnModificar2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Modificar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminar2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCodigo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnrestaurar2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Restaurar</string>
    </property>
   </widget>
   <zorder>txtCodigo</zorder>
   <zorder>label</zorder>
   <zorder>tablaclientes</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>btnGuardar</zorder>
   <zorder>btnModificar2</zorder>
   <zorder>btnEliminar2</zorder>
   <zorder>btnrestaurar2</zorder>
  </widget>
 </widget>
 <resources>
  <include location="imagenes/imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
